<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>566</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="box_brilho">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_B">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>271</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>270</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BRILHO</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_E">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-12</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_exposicao">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-12</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-12</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_S">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_C">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>421</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_saturacao">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>371</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="box_contraste">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>421</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_S">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SATURAÇÃO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_E">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXPOSIÇÃO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONTRASTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_I">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_webcam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_webcam">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Webcam</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>box_brilho</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_E</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>box_exposicao</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_S</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>box_saturacao</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_C</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>box_contraste</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_brilho</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_B</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_exposicao</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_E</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_saturacao</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_S</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_contraste</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_C</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
